apps/api/src/domain/models/context/commands/connect-resources-with-note/connect-resources-with-note.command.spec.ts

```diff
@@ -0,0 +1,582 @@
+                                initialState: initialStateWithAllResourcesButNoConnections,
+                            });
+                        });
+                    });
+                });
+        });
+
+        describe(`when the resource type is a non-resource aggregate type on the property`, () => {
+            ['toCompositeIdentifier', 'fromCompositeIdentifier'].forEach((propertyName) =>
+                describe(propertyName, () => {
+                    Object.values(AggregateType)
+                        .filter((t) => !isResourceType(t))
+                        .forEach((invalidResourceType) => {
+                            describe(`when the resource type is: ${invalidResourceType}`, () => {
+                                it(`should fail with a type error`, async () => {
+                                    await assertCommandFailsDueToTypeError(
+                                        assertionHelperDependencies,
+                                        {
+                                            propertyName: propertyName,
+                                            invalidValue: {
+                                                type: invalidResourceType,
+                                                id: buildDummyUuid(567),
+                                            },
+                                        },
+                                        commandFsaFactory.build(buildDummyUuid(662))
+                                    );
+                                });
+                            });
+                        });
+                })
+            );
+        });
+
+        describe(`when the context for the a member has an unregistered context type`, () => {
+            const bogusContextType = 'totally-booogus8';
+
+            ['toMemberContext', 'fromMemberContext'].forEach((propertyName) => {
+                describe(`${propertyName} : ${bogusContextType}`, () => {
+                    it(`should fail with the expected error`, async () => {
+                        await assertCreateCommandError(assertionHelperDependencies, {
+                            systemUserId: dummySystemUserId,
+                            initialState: initialStateWithAllResourcesButNoConnections,
+                            buildCommandFSA: (id) =>
+                                commandFsaFactory.build(id, {
+                                    [propertyName]: { type: bogusContextType },
+                                }),
+                        });
+                    });
+                });
+            });
+        });
+
+        describe(`when the to member does not exist`, () => {
+            it(`should fail with the expected error`, async () => {
+                await assertCreateCommandError(assertionHelperDependencies, {
+                    systemUserId: dummySystemUserId,
+                    initialState: new DeluxeInMemoryStore({
+                        // to member DNE
```

Comment:
This is to come here?

Reply:
No. DNE = "Does Not Exist". I'll reword this. I'm just being explicit about how I am invalidating the state for this test case.

Reply:
done